main/adapters.py

```diff
@@ -13,6 +19,37 @@ def save_user(self, request, sociallogin, form=None):
         else:
             new_user = UserProfile(user=user, name=user.get_full_name())
             new_user.save()
-        # print("Inside the adapter")
         return redirect('/')
         
+class AccountAdapter(DefaultAccountAdapter):
```

Comment:
For this AccountAdapter i suggest you to go the sms19-env/lib/python3.7/site-packages/allauth/account/adapters.py file.  go through line 217 to 244 . Much of the code here is copy paste from that . Just some lines have been changed for the field 'name'  .